component/loki/source/kubernetes_events/event_controller.go

```diff
@@ -96,9 +96,12 @@ func (ctrl *eventController) runError(ctx context.Context) error {
 		return fmt.Errorf("adding core to scheme: %w", err)
 	}
 
+	// TODO: how should we handle this?
+	defaultNamespaces := map[string]cache.Config{}
+	defaultNamespaces[ctrl.task.Namespace] = cache.Config{}
 	opts := cache.Options{
-		Scheme:     scheme,
-		Namespaces: []string{ctrl.task.Namespace},
+		Scheme:            scheme,
+		DefaultNamespaces: defaultNamespaces,
```

Comment:
They changed the list to a map where we can define a specific config. What should we do about it?

Reply:
Here is the change: https://github.com/kubernetes-sigs/controller-runtime/commit/3e35cab1ea29145d8699259b079633a2b8cfc116